src/main/31-cloudfront.tf

```diff
@@ -16,6 +20,24 @@ module "fe_cdn" {
       domain_name           = module.fe_s3_bucket.s3_bucket_bucket_domain_name
       origin_access_control = "fe_hosting_oac"
     }
+
+    apigw = {
+      origin_id   = var.api_gateway_origin_id
+      domain_name = "${aws_api_gateway_rest_api.apigw.id}.execute-api.${var.aws_region}.amazonaws.com"
```

Comment:
Use the`invoke_url` attribute from the APIGW stage. Note that if you use that you won't need `origin_path` here